docs/manage/config-policies.rst

```diff
@@ -0,0 +1,263 @@
+.. _config-policies:
+
+#######################
+ Config Policies Guide
+#######################
+
+Config Policies allow administrators to set limits on how users can define workloads (e.g.,
+experiments, notebooks, tensorboards, shells, and commands). Administrators can include any
+parameter referenced in the :ref:`experiment configuration reference <experiment-config-reference>`.
+
+.. include:: ../_shared/attn-enterprise-edition.txt
+
+Config Policies allow admins to define two types of configurations:
+
+-  **Invariant Configs**: Settings applied to all workloads within a scope (global or workspace).
+-  **Constraints**: Restrictions that prevent users from exceeding resource limits.
+
+These policies are essential for managing task priorities, resource allocation, and setting specific
+configurations across the organization. Global policies override workspace policies, and both
+override user-submitted configurations.
+
+**************
+ Key Features
+**************
+
+-  Apply policies via WebUI or CLI
+-  Define limits for resource usage, environment settings, and more
+-  Apply policies at different levels (cluster-wide or workspace)
+-  Override capabilities for specific user roles or groups
+-  Set different priority limits for experiments and NTSC (notebooks, TensorBoards, shells, and
+   commands) tasks
+-  Allow priority overrides within policy constraints
+
+******************
+ Setting Policies
+******************
+
+Administrators can set Config Policies at either the cluster or workspace levels through the WebUI
+or CLI.
+
+WebUI
+=====
+
+Administrators can set Config Policies at both the cluster and workspace levels.
+
+.. tabs::
+
+   .. tab::
+
+      Cluster Config Policy
+
+      To set configuration policies at the cluster level:
+
+      #. Sign in to the Determined WebUI as a cluster administrator.
+      #. Navigate to **Config Policies**.
+      #. Choose **Experiments** or **Tasks** to display the editable configuration file.
+      #. Define the policies, then click **Apply**.
+      #. Confirm you want to apply the policies.
+
+   .. tab::
+
+      Workspace Config Policy
+
+      To set configuration policies at the workspace level:
+
+      #. Sign in to the Determined WebUI as a cluster or workspace administrator.
+      #. From your workspace, navigate to the **Config Policies** tab.
+      #. Choose **Experiments** or **Tasks** to display the editable configuration file.
+      #. Define the policies, then click **Apply**.
+      #. Confirm you want to apply the policies.
+
+CLI
+===
+
+Use the following commands to manage Task Config Policies via CLI:
+
+.. code:: bash
+
+   # List existing policies
+   det policy list
```

Comment:
It's `det config-policies`

```
% det config-policies -h
usage: det config-policies [-h] subcommand ...

positional arguments:
  subcommand
    help      show help for this command
    delete    delete config policies
    describe  describe config policies
    set       set config policies
``` 

I'm getting confirmation on exact commands, i.e. how to specify global vs. workspace.

Reply:
i understand there no CLI commands for setting config policies at the global level, so i'll fix the CLI section.